### PR TITLE
Add ChefDeprecations/VerifyPropertyUsesFileExpansion cop

### PR DESCRIPTION
**Pull request by tas50**
Detects incorrect verify property usage that breaks Chef 13+

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
Resolves #95

File: lib/rubocop/cop/chef/deprecation/verify_property_file_expansion.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        #
+        #   # good
+        #   file '/etc/nginx.conf' do
+        #     verify 'nginx -t -c %{path}'
+        #   end
+        #
+        class VerifyPropertyUsesFileExpansion < Cop
+          include RuboCop::Chef::CookbookHelpers
+
+          MSG = "Use the 'path' property in the verify property and not the 'file' property which was removed in Chef Infra Client 13.".freeze
```
Comment (tyler-ball) [inline lib/rubocop/cop/chef/deprecation/verify_property_file_expansion.rb]: `'path' property` -> `'path' variable`

Comment (tyler-ball) [inline lib/rubocop/cop/chef/deprecation/verify_property_file_expansion.rb]:
```suggestion
          MSG = "Use the 'path' variable in the verify property and not the 'file' property which was removed in Chef Infra Client 13.".freeze
```

Comment (tas50) [reply lib/rubocop/cop/chef/deprecation/verify_property_file_expansion.rb]: Nice catch. I fixed it in the docs and cop

Comment (tas50) [reply lib/rubocop/cop/chef/deprecation/verify_property_file_expansion.rb]: Fixed here too

**Comment by tyler-ball** [comment]
tenor-173700498

**Pull request merged**